### PR TITLE
Add confirmation to Update dialogue

### PR DESCRIPTION
**Pull request by stef-the**
I have completed my task.

### CODE REVIEW

**Comment by BotchedRPR** [comment]
This fixes #4

File: .gitignore [hunk 1]
```diff
@@ -128,4 +128,9 @@ $RECYCLE.BIN/
 # Mac desktop service store files
 .DS_Store
 
-_NCrunch*
\ No newline at end of file
+_NCrunch*
+launcher/obj/project.nuget.cache
```
Comment (BotchedRPR) [inline .gitignore]: Unnecessary. Gitignore already has obj in it, You just need to delete the folder from the tree.

File: launcher/UpdateWindow.xaml.cs [hunk 2]
```diff
@@ -28,6 +28,18 @@ public UpdateWindow()
...
         {
+        }
+        private void HandleProgress(object sender, DownloadProgressChangedEventArgs e)
+        {
+            UpdateBar.Value = e.ProgressPercentage;
+        }
+
+        private async void Confirm_Button_Click(object sender, RoutedEventArgs e)
+        {
+            Label Question0 = (Label)this.FindName("Question0");
```
Comment (BotchedRPR) [inline launcher/UpdateWindow.xaml.cs]: Couldn't you have just used Question0.Visibility etc?

File: launcher/Updater.cs [hunk 3]
```diff
@@ -21,23 +22,23 @@ public static async void CheckForUpdate()
 
                 //Setup the versions
                 Version latestGitHubVersion = new Version(releases[0].TagName);
-                Version localVersion = new Version("1.0.1"); //Replace this with your local version. 
+                Version localVersion = new Version("1.0.0"); //Replace this with your local version.
```
Comment (BotchedRPR) [inline launcher/Updater.cs]: Change this to 1.0.4

**Review by BotchedRPR — CHANGES_REQUESTED** [review]

File: launcher/UpdateWindow.xaml.cs (hunk 2 above)
Comment (Neintonine) [reply launcher/UpdateWindow.xaml.cs]: yes. intilizeComponent does create those properties already.

Comment (BotchedRPR) [reply launcher/UpdateWindow.xaml.cs]: both methods are fine but I think that this one just adds more clutter

Comment (stef-the) [reply launcher/UpdateWindow.xaml.cs]: done

File: .gitignore (hunk 1 above)
Comment (stef-the) [reply .gitignore]: removed what I added to gitignore

File: launcher/Updater.cs (hunk 3 above)
Comment (stef-the) [reply launcher/Updater.cs]: also done

**Comment by stef-the** [comment]
this should be ready? not 100% sure

**Comment by BotchedRPR** [comment]
wait why is this the main branch?!

**Review by BotchedRPR — APPROVED** [review]
all good

**Pull request merged**